src/Fantomas.Core/ASTTransformer.fs

```diff
@@ -2975,7 +2975,10 @@ let mkModuleSigDecl (creationAide: CreationAide) (decl: SynModuleSigDecl) =
         )
         |> ModuleDecl.Exception
     | SynModuleSigDecl.ModuleAbbrev(ident, lid, StartRange 6 (mModule, _)) ->
-        ModuleAbbrevNode(stn "module" mModule, mkIdent ident, mkLongIdent lid, declRange)
+        let alias = mkLongIdent lid
```

Comment:
Should we leave a comment here that this can be revisited after https://github.com/dotnet/fsharp/issues/14912 is fixed?

Reply:
Let's just fix it this week and integrate it.